presto-orc/src/main/java/com/facebook/presto/orc/reader/TimestampBatchStreamReader.java

```diff
@@ -202,40 +203,6 @@ public String toString()
                 .toString();
     }
 
-    // This comes from the Apache Hive ORC code
-    public static long decodeTimestamp(long seconds, long serializedNanos, long baseTimestampInSeconds)
```

Comment:
these methods are moved to a TimestampUtils class as these are used both by the Selective and Batch readers

Reply:
Could you move this change into a separate commit?

Reply:
will make a new commit for this